ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -7,4 +7,4 @@ ref:
       cpu: 1000m
       memory: 2Gi
   documentation: |-
-    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform. Anything that needs to be configured using manifests should generate them before this step and put them in the SHARED_DIR with the filename manifest_(manifest_name).yml so that this step can pull in the manifest in. 
+    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform. Anything that needs to be configured using manifests should generate them before this step and put them in the SHARED_DIR with the filename manifest_(manifest_name).yml so that this step can pull in the manifest in.
```

Comment:
I like dropping trailing whitespace, but it is orthogonal to your PR, so I'd rather drop this here.

Reply:
OK